### DISCUSSION TITLE
Interpreting Cohen's d Effect Size visualisation - can't get slider settings to change in Settings

### DISCUSSION THREAD

**Discussion by JCScan** [rpsychologist/rpsychologist-com#38] [Q&A] [answered]
(I apologise in advance if this is nearly the dumbest question ever)
I'm doing Daniel Lakens' Statistical Inferences course on Coursera and in Week4 he's using your Interpreting Cohen's d Effect Size visualisation in one of the assignments. First off, under slider settings, he wants us to change Max to 0.002 and Step Size to 0.001. I do this, then press the 'Save settings to browser' button and get a green tick. However going to the visualisation, nothing has changed and I just can't get it to work. I've tried another browser (Edge; I'm using Chrome as default). I've turned off any blocking on the page. Java is enabled. I note Daniel pointed to a much older URL (http://rpsychologist.com/d3/cohend) and maybe you've changed the minimum values allowed. Anyway, grateful for any advice. BTW it's a superb visualisation.

**Comment by rpsychologist** [comment, marked as answer]
Thanks for posting the question here! This is common a question, that's caused by a dumb design choice by me.

The "save settings to brower" doesn't actually update the values, it only saves to current settings so that they persist the next time you visit the site. You need to hit enter to update the value. Currently, unsubmitted fields are shown in a slightly different color, which is extremely vague.
![image](https://user-images.githubusercontent.com/1629860/156149532-8f09068c-8ca1-41db-a231-df407c2296b4.png)

There's an open issue for this https://github.com/rpsychologist/rpsychologist-com/issues/8, I really should prioritize fixing this ASAP.

**Reply by JCScan** [reply]
Thank you very kindly. Of course worked a breeze.

**Comment by drarifozer** [comment]
Hi
When Cohen d =0, there is an explanation like that: "Moreover, in order to have one more favourable outcome in the "treatment" group compared to the "control" group, we need to treat 16.5 people on average. " it is seen as NNT= 16.5 on the graph. Then again, we calculate the number of people in the experimental group, ex. 20+6.1. This is confusing, at least mine :).
Is 16.5 a raw number or a percentage, because the numbers you use in the whole process are percentages, not raw points.
Thank you


ARR=(Control event rate)− (Experimental event rate) 
ARR=0.2−0.3651=−0.1651
NNT=1/ARR
NNT=1/−0.1651=−6.1
